thorlcr/master/thmastermain.cpp

```diff
@@ -1158,6 +1164,10 @@ int main( int argc, const char *argv[]  )
 #endif
         configurePreferredPlanes();
 
+        assertex(startupTimeTracker.numInFlight() == 0);
+        __uint64 startupElapsedTimeNs = startupTracker.elapsedNs();
+        recordGlobalMetrics("Queue", {{"component","thor"},{"name",thorName}}, { StNumStarts, StTimeStart, StTimeStartElapsed }, { 1ULL, startupTimeTracker.getWaitingNs()+startupElapsedTimeNs, startupElapsedTimeNs });
```

Comment:
looks like this is double counting the connect() time.. ? 
startupTimeTracker is connect() time, startupTracker is all time inc. connect time?

startupTimeTracker vs startupTracker could do with better names.